src/v2/models/AppState.js

```diff
@@ -33,6 +33,7 @@ export default Model.extend({
     remediations: 'array',
     dynamicRefreshInterval: 'number',
     deviceFingerprint: 'string',
+    updatePollingCancelLink: 'boolean',
```

Comment:
change this too `updateFooterLink`

Reply:
instead of this.. did you explore triggering an event from `onPollingFail` ??

this.trigger('updateFooterLink');